test/component/scheduler/perf/test-performance.sh

```diff
@@ -34,13 +37,10 @@ trap cleanup EXIT
 kube::etcd::start
 kube::log::status "performance test start"
 
-# TODO: set log-dir and prof output dir.
-DIR_BASENAME=$(basename `pwd`)
-go test -c -o "${DIR_BASENAME}.test"
 # We are using the benchmark suite to do profiling. Because it only runs a few pods and
 # theoretically it has less variance.
-"./${DIR_BASENAME}.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
+go test -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
```

Comment:
@wojtek-t Yes. I believe he means so. Without the binary, we cannot analyze the output.